src/managers/AutoModerationRuleManager.js

```diff
@@ -74,6 +74,8 @@ class AutoModerationRuleManager extends CachedManager {
    * @typedef {Object} AutoModerationActionMetadataOptions
    * @property {GuildTextChannelResolvable|ThreadChannel} [channel] The channel to which content will be logged
    * @property {number} [durationSeconds] The timeout duration in seconds
+   * @property {string} [customMessage]
+   * The custom message that will be shown to members whenever the rule blocks their message.
```

Comment:
This description doesn't match the string in main.

Reply:
it was copied from ddevs as i remember

Reply:
You shouldn't be doing that, especially if you call this pull request a backport.